### PR TITLE
v1.3.0

### PR DESCRIPTION
**Pull request by JudithBernett**
- New support for leave-tissue-out (LTO) split
- Included NaiveTissueMeanPredictor and ProteomicsRandomForest
- Fixed an error for early splitting for when there are less than 4 groups in validation
- MultiOmicsNN: changed loading of features to match dataloader 
- Lots of restructuring in visualization
- Kicked out partial correlation. Instead normalized metrics.
- Updated Readme
- New package versions
- New safety scan

### CODE REVIEW

File: docs/conf.py
Comment (JudithBernett) [inline docs/conf.py:1]: 1.3.0 instead because of LTO feature

File: drevalpy/models/baselines/naive_pred.py [hunk 1]
```diff
@@ -331,11 +331,119 @@ def load_cell_line_features(self, data_path: str, dataset_name: str) -> FeatureD
...
+        Loads the drug features.
+
+        :param data_path: Path to the data.
+        :param dataset_name: Name of the dataset.
+        :return: FeatureDataset containing the drug IDs.
+        """
+        return load_drug_ids_from_csv(data_path, dataset_name)
+
+
+class NaiveTissueMeanPredictor(DRPModel):
```
Comment (JudithBernett) [inline drevalpy/models/baselines/naive_pred.py:343]: add in documentation

File: drevalpy/models/baselines/sklearn_models.py [hunk 2]
```diff
@@ -242,3 +242,33 @@ def build_model(self, hyperparameters: dict):
             learning_rate=hyperparameters.get("learning_rate", 0.1),
             max_depth=hyperparameters.get("max_depth", 3),
         )
+
+
+class ProteomicsRandomForest(RandomForest):
```
Comment (JudithBernett) [inline drevalpy/models/baselines/sklearn_models.py:247]: add in documentation

**Review by JudithBernett — COMMENTED** [review]
some open fixes

**Review by PascalIversen — APPROVED** [review]
